doc/man1/openssl-verify.pod.in

```diff
@@ -499,7 +502,8 @@ certificate of an untrusted certificate cannot be found.
 =item B<X509_V_ERR_UNABLE_TO_VERIFY_LEAF_SIGNATURE>
 
 No signatures could be verified because the chain contains only one
-certificate and it is not self signed.
+certificate and it is not self-signed
+(nor self-issued in case the B<-partial_chain> option is set).
```

Comment:
I've improved this to
```
No signatures could be verified because the chain contains only one certificate
and it is not self-signed and the B<-partial_chain> option is not used.

```
and analogously in `X509_STORE_CTX_get_error.pod`.